datajunction-ui/src/app/services/DJService.js

```diff
@@ -146,7 +153,7 @@ export const DataJunctionAPI = {
     metricSelection.map(metric => params.append('metrics', metric));
     dimensionSelection.map(dimension => params.append('dimensions', dimension));
     const data = await (
-      await fetch(DJ_URL + '/data/?' + params + '&limit=100')
+      await fetch(DJ_URL + '/data/?' + params + '&limit=100&async_=true')
```

Comment:
I see this was switched to async but I don't see polling anywhere. Isn't some kind of polling mechanism required to eventually display the data after the "Run Query" button is clicked on the SQL page?

Reply:
Good point! Let me fix this